### PR TITLE
Apptentive doc updates

### PR DESCRIPTION
**Pull request by melj32app**
Refresh product capabilities and provide further details.

# Summary

(Please provide a high level summary)

### CODE REVIEW

**Comment by melj32app** [comment]
@cla-bot check

File: src/pages/integrations/apptentive/event/index.md [hunk 1]
```diff
@@ -2,37 +2,46 @@
...
-* User feedback
-* App events
-* eCommerce events
+* Custom Events
+* User Attributes
 
 ## Prerequisites
 
-In order to enable mParticle's integration with Apptentive you will need an Apptentive account to obtain your Apptentive App Key and Apptentive App Signature. These are available from the API & Development page in the Apptentive Dashboard.
+In order to enable mParticle’s integration with Apptentive, you will need your Apptentive App Key and Apptentive App Signature for each respective app and environment which you can get from Apptentive's [API and Development](https://be.apptentive.com/apps/current/settings/api) page. If you need assistance to get an account, you can contact your Apptentive representative or reach out [here](mailto:support@apptentive.com).
```
Comment (samdozor) [inline src/pages/integrations/apptentive/event/index.md:36]:
```suggestion
In order to enable mParticle's integration with Apptentive, you will need your Apptentive App Key and Apptentive App Signature for each respective app and environment which you can get from Apptentive's [API and Development](https://be.apptentive.com/apps/current/settings/api) page. If you need assistance to get an account, you can contact your Apptentive representative or reach out [here](mailto:support@apptentive.com).
```

File: src/pages/integrations/apptentive/event/index.md [hunk 2]
```diff
@@ -42,47 +51,74 @@ target '<Your Target>' do
 end
 ~~~
 
-~~~java
+~~~Java
 //Sample build.gradle
 dependencies {
     compile ('com.mparticle:android-apptentive-kit:4.+')
```
Comment (samdozor) [inline src/pages/integrations/apptentive/event/index.md:57]:
```suggestion
    compile ('com.mparticle:android-apptentive-kit:5.+')
```

**Review by samdozor — CHANGES_REQUESTED** [review]

**Comment by samdozor** [comment]
@melj32app thank you so much for the PR. Looks great. I made two small tweaks but otherwise this looks good. Can you go ahead and sign the cla per the note above? then we'll get this merged.

**Comment by melj32app** [comment]
@samdozor I believe it is saying that you aren't cla-signed as I did the cla check and comment.

**Comment by melj32app** [comment]
@cla-bot check

**Review by samdozor — APPROVED** [review]

**Pull request merged**